### DISCUSSION TITLE
Is there a way to access the query params attached to the workload item parent window url

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/Microsoft-Fabric-workload-development-sample#167] [Help]
I am trying to access the workload item url directly to get a sharable link directly from inside the workload item and also I want to attach a set of query parameters to it and use them in my workload item, later when someone uses the same link. How can we do this trough the fabric-workload client.

**Comment by IdanGoldfarb1989** [comment]
Hi, no at the moment this is not possible.
